README.md

```diff
@@ -19,65 +42,34 @@ Mautic Introduction
-
-<table width="100%" border="0">
-	<tr>
-		<td>
-			<center><b>Step 1</b></center>
-		</td>
-		<td>
-			<center><b>Step 2</b></center>
-		</td>
-		<td>
-			<center><b>Step 3</b></center>
-		</td>
-	</tr>
-	<tr>
-		<td align="center" width="33.3%">
-			<a href="https://github.com/mautic/mautic/archive/master.zip">Download the repository zip</a><br />Extract this zip to your web root.
-		</td>
-		<td align="center" width="33.3%">
-			Run the following command to install required packages.<br /> <code>composer install</code>
-		</td>
-		<td align="center" width="33.3%">
-			Open your browser and complete the installation through the web installer.
-		</td>
-	</tr>
-</table>
+`*`Security support for 2.16 will only be provided for Mautic itself, not for core dependencies that are EOL, such as Symfony 2.8.
 
-**Get stuck?** *No problem. Check out [general troubleshooting](https://docs.mautic.org/en/troubleshooting) and if it won't solve your issue join us at the <a href="https://www.mautic.org/community">Mautic community</a> for help and answers.*
+## Software Downloads
+The GitHub version is recommended for both development and testing. The production package (including all libraries) is available at [mautic.org/download](https://www.mautic.org/download).
 
-## Disclaimer
-Installing from source is only recommended if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and to keep it working. If the source and/or database schema gets out of sync with Mautic's releases, the release updater may not work and will require manual updates. For production the pre-packaged Mautic available at [mautic.org/download](https://www.mautic.org/download) is recommended.
+## Installation
+### Disclaimer
+*Install from source only if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and keep it working. If the source/database schema gets out of sync with Mautic releases, the release updater may not work and will require manual updates. For production, we recommend the pre-packaged Mautic which is available at [mautic.org/download](https://www.mautic.org/download).*
 
-*Also note that the source outside <a href="https://github.com/mautic/mautic/releases">a tagged release</a> should be considered "alpha" and may contain bugs, cause unexpected results, data corruption or loss, and is not recommended for use in a production environment. Use at your own risk.*
+*Also note that source code outside of a [tagged release](https://github.com/mautic/mautic/releases) should be considered ‘alpha’. It may contain bugs, cause unexpected results, data corruption or loss, and is not recommended for use in a production environment. Use at your own risk.*
```

Comment:
Refactor for link macro

```suggestion
*Also note that source code outside of a [tagged release][tagged-release] should be considered ‘alpha’. It may contain bugs, cause unexpected results, data corruption or loss, and is not recommended for use in a production environment. Use at your own risk.*
```